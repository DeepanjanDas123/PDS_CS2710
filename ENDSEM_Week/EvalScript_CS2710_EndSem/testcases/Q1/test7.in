BuildTreeFromFrequencyTable
62
v 25
E 36
6 43
_ 3
5 31
d 12
N 47
a 40
8 30
1 22
W 50
e 14
J 2
P 15
l 39
X 33
K 31
Z 19
o 16
h 35
p 38
g 37
L 38
i 6
O 23
T 41
f 46
t 25
R 4
k 17
2 50
n 12
C 28
w 45
S 8
q 11
c 16
M 44
G 45
z 10
V 12
3 25
x 28
9 15
s 17
U 7
j 19
u 39
b 21
H 14
0 34
y 15
7 19
Q 1
r 34
D 10
I 17
m 22
Y 22
A 19
B 20
F 18
Quit
