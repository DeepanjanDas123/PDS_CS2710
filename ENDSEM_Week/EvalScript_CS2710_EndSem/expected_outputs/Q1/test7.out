(((((E,g),(((S,D),7),L)),((p,(A,Z)),(l,u))),((((j,B),a),(T,((z,(R,i)),b))),((6,M),((1,Y),G)))),((((w,(m,O)),(f,N)),((((q,V),(d,n)),2),(W,(3,t)))),((((v,(((_,(Q,J)),U),H)),(C,x)),(((e,9),8),((P,y),5))),(((K,(c,o)),(X,0)),((r,(I,k)),(h,(s,F)))))));
